lib/tapioca/generator.rb

```diff
@@ -116,16 +116,26 @@ def build_todos
       say("Please review changes and commit them.", [:green, :bold])
     end
 
-    sig { params(requested_constants: T::Array[String]).void }
-    def build_dsl(requested_constants)
+    sig do
+      params(
+        requested_constants: T::Array[String],
+        should_verify: T::Boolean,
+      ).void
+    end
+    def build_dsl(requested_constants, should_verify: false)
       load_application(eager_load: requested_constants.empty?)
       load_dsl_generators
 
-      rbi_files_to_purge = existing_rbi_filenames(requested_constants)
-
-      say("Compiling DSL RBI files...")
+      if should_verify
+        say("Checking for out-of-date RBIs...")
+      else
+        say("Compiling DSL RBI files...")
+      end
       say("")
 
+      outpath = should_verify ? Dir.mktmpdir : config.outpath
```

Comment:
Are we not cleaning up after ourselves by removing the tmp directory?

Reply:
Good point. I'll double check, previously when we were using the block syntax in a different method, it cleaned up automatically after the block executed. May need to manually do that. I'll double check and update if necessary.